core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -670,6 +670,21 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey UNDERFS_OBJECT_STORE_DIRECT_PERSIST_ENABLED =
```

Comment:
If this is an optimization (but unsafe), then why is it isolated to object stores? Why can't any ufs use this mode?

Reply:
currently, we check the ufs type. in case it is object store and this flag is true, we skip the tmp files; otherwise temp files are still used. Are you suggesting to enable this regardless of ufs type?

Reply:
Yeah, I'm asking why this cannot be possible for any persist job, instead of only for object stores?